src/matter/cluster/LevelControlCluster.ts

```diff
@@ -0,0 +1,179 @@
+/**
+ * @license
+ * Copyright 2022 The node-matter Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { OptionalAttribute, WritableAttribute, OptionalWritableAttribute, Attribute, Cluster, OptionalCommand, TlvNoResponse } from "./Cluster";
+import { BitFlag, MatterApplicationClusterSpecificationV1_0, TlvBitmap, TlvEnum, TlvField, TlvNullable, TlvObject, TlvUInt16, TlvUInt8 } from "@project-chip/matter.js";
+
+/** @see {@link MatterApplicationClusterSpecificationV1_0} § 1.6.5.8 */
+export const OptionsBitmap = {
+    /** Command execution control - see 1.6.5.8.1. */
+    ExecuteIfOff: BitFlag(0), // TODO default: true.
```

Comment:
This should start be a lowercase letter since this will be the name of the flag when using it.
Bitmap definition will be used to create a flag object:
{
    flag1: boolean,
    flag2: boolean,
}

Same below